src/app/cocktail/cocktail.component.ts

```diff
@@ -1,10 +1,15 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, ViewEncapsulation } from '@angular/core';
 
 @Component({
   selector: 'app-cocktail',
   templateUrl: './cocktail.component.html',
-  styleUrls: ['./cocktail.component.css']
+  styleUrls: ['./cocktail.component.css'],
+  encapsulation: ViewEncapsulation.None
 })
 export class CocktailComponent {
   @Input() cocktail;
+
+getImageUrl (cocktail) {
+  return `http://${cocktail.strDrinkThumb}`;
```

Comment:
on Heroku we have the problem with mixed content type. When we call API via http protocol on page with https protocol. So will be great to remove protocol from this method: ```return `//${cocktail.strDrinkThumb}`;```